docs/book/src/clusterctl/developers.md

```diff
@@ -157,7 +156,7 @@ Optionally, you may want to check if the components are running properly. The
 exact components are dependent on which providers you have initialized. Below
 is an example output with the docker provider being installed.
 
-```
+```bash
```

Comment:
Can you split `kubectl get deploy -A | grep "cap\|cert"` from the output

Reply:
Done